### DISCUSSION TITLE
Editing Reports

### DISCUSSION THREAD

**Discussion by tbezo** [qatrackplus/qatrackplus#577] [Q&A]
Hi,

I would like to edit/change parts of the default report templates (e.g. add a logo). Where would be a good starting point.

regards
Thomas

**Comment by randlet** [comment]
The main template for generating the reports are here:  [html](https://github.com/qatrackplus/qatrackplus/blob/master/qatrack/reports/templates/reports/html_report.html) [pdf](https://github.com/qatrackplus/qatrackplus/blob/master/qatrack/reports/templates/reports/pdf_report.html). Those includes a few other templates (e.g. `_header.html` and `_details.html`). 

Note for pdf reports, if you make changes to css files or add images, you need to run `python manage.py collectstatic` before those changes will be visible in the pdf.
